tests/TestHarness/Cluster.py

```diff
@@ -262,17 +259,15 @@ def launch(self, pnodes=1, unstartedNodes=0, totalNodes=1, prodCount=1, topo="me
                 return False
             tries = tries - 1
             time.sleep(2)
-
-        cmd="%s -p %s -n %s -d %s -i %s -f %s --unstarted-nodes %s" % (
-            "python3 tests/launcher.py",  
-            pnodes, totalNodes, delay, 
-            datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3],
-            producerFlag, unstartedNodes)
+        loggingLevelDictString = json.dumps(self.loggingLevelDict, separators=(',', ':'))
+        cmd="%s -p %s -n %s -d %s -i %s -f %s --unstarted-nodes %s --logging-level %s --logging-level-map %s" % (
+            "python3 tests/launcher.py", pnodes, totalNodes, delay, datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3],
+            producerFlag, unstartedNodes, self.loggingLevel, loggingLevelDictString)
         cmdArr=cmd.split()
         if self.staging:
             cmdArr.append("--nogen")
 
-        nodeosArgs="--max-transaction-time -1 --abi-serializer-max-time-ms 990000 --p2p-max-nodes-per-host %d --max-clients %d" % (maximumP2pPerHost, maximumClients)
+        nodeosArgs="--resource-monitor-not-shutdown-on-threshold-exceeded --max-transaction-time -1 --abi-serializer-max-time-ms 990000 --p2p-max-nodes-per-host %d --max-clients %d" % (maximumP2pPerHost, maximumClients)
```

Comment:
@heifner  If I remember right I added this at some point and we removed it. Is that correct, or are we okay with it now?

Reply:
https://github.com/AntelopeIO/leap/commit/25768770c3f56484ee3fe34c1d13739da95408da

Reply:
Following a discussion, this should be put behind a version check but is otherwise fine.

Reply:
Removed again, due to the overhead of creating a test.